src/natcap/invest/urban_cooling_model.py

```diff
@@ -935,6 +950,8 @@ def calculate_energy_savings(
     target_building_vector = gdal.OpenEx(
         target_building_vector_path, gdal.OF_VECTOR | gdal.GA_Update)
     target_building_layer = target_building_vector.GetLayer()
+    target_building_srs = target_building_layer.GetSpatialRef()
+    target_building_square_units = target_building_srs.GetLinearUnits() ** 2

```

Comment:
I think I saw that the building vectors are reprojected to the LULC. But just wanted to make sure, otherwise `GetLinearUnits` on a Geographic vector could get weird.

Reply:
Yep!  This is a really important point to make, and you're right that the buildings vector is reprojected to the projection of the LULC (task `intermediate_building_vector_task`), which validation enforces as being projected in `m`.  I've added notes to the docstrings of `execute` (about the LULC being in a linearly-projected CRS) and to `calculate_energy_savings` (about the need for the buildings vector being in a linearly-projected CRS) to clarify the contract.  Patched in 05a0b834